gravitee-am-management-api/gravitee-am-management-api-service/src/main/java/io/gravitee/am/management/service/impl/plugins/BotDetectionPluginServiceImpl.java

```diff
@@ -100,4 +103,12 @@ private BotDetectionPlugin convert(Plugin plugin) {
         botDetectionPlugin.setFeature(plugin.manifest().feature());
         return botDetectionPlugin;
     }
+
+    public Completable checkPluginDeployment(String type) {
```

Comment:
If I'm not mistaken, this method is doing exactly the same that is done by the extended class